### ISSUE TITLE
Filter functionality use

### ISSUE THREAD

**Issue by suzan-uskudarli** [bnbcfyh/cmpe352-2021-repeat-project#13]
When attempting to filter events by date i get nothing. I can not tell if there are no events, if the date format is wrong or what may be the issue.

It is better to give feedback in results such as:

(0) tennis events are found between 15.07.2021 and 16.09.2021 

That way it is clear that the date format is corret and that the query is understood.

**Comment by bnbcfyh** [comment]
I agree, an explaining feedback ought to be given to the user. The results might be listed as given in this issue's description. The implementation is pending.
